NHLBI_BioData_Catalyst/README.md

```diff
@@ -30,7 +48,7 @@ To get your token, process as follows:
 3. On the pop-up window, click on REFRESH and then COPY
 4. Back into your Jupyter environment, create a text file called `token.txt` into the python and/or R folders and paste the token into it.
```

Comment:
I would add a specific filepath here to describe where to place the token.txt file, just to avoid confusion